features/fixtures/storeHealthCertificate/StoreHealthCertificate-response.json

```diff
@@ -0,0 +1,32 @@
+{
```

Comment:
The naming of this file is a bit misleading since this is not actually the response to the Store Health Card transaction ([defined here](https://worldhealthorganization.github.io/ddcc/transactions.html#register-health-certificate-message-semantics-response)). This is kind of wordy, but maybe something like `retrieveStoreHealthCertificateResults-response.json`, or something showing that this isn't the response from a Store Health Certificate transaction but actually from retrieving one that has already been stored.